resources/questions/basic_sails_and_sailing_1.yml

```diff
@@ -0,0 +1,6 @@
+question: |
+  Бегучий такелаж (running rigging) – это совокупность канатов и шкотов, используемая для регулировки угла парусов к ветру
+options:
+  - Правда
+  - Неправда
+correct: Правда
```

Comment:
why no newline?

Reply:
ok, you got a couple of q with newlines at EOF, and part - without.
why?)

Reply:
No reason, just random)
Will be fixed to one style

Reply:
fixed